### ISSUE TITLE
Computer hates me.

### ISSUE THREAD

**Issue by dorothealint** [dorothealint/Rick-and-Morty#8]
...as you're well aware by now.

I attempted to commit the short scripts I created with years and characters and it is giving me a fatal error.  Tells me to git push and when I do it doesn't want what I'm pushing. LOL.  So...the short scripts are in the existdb file folder you created on Wednesday but they are floating in Nowheresville on planet github so they aren't on there.
@quantum-satire

**Comment by helvitiis** [comment]
@dorothealint Dang it! Can you send me screenshots of what's happening? Let's try to debug...

**Comment by dorothealint** [comment]
I think I fixed it.  It was with github itself.  It let me pull everything just fine but I had to reset my password (to the same thing it was before) and log into all the different directories I have again to be able to push anything.  I believe it worked for me for all three directories now!

**Comment by helvitiis** [comment]
That's so odd it did that. Are you connecting to GitHub through a different WiFi connection than usual?

**Comment by dorothealint** [comment]
Nope.  I'm at home.  Sometimes my wifi slows down to the point I have to reset the modem and router though.  It's Armstrong cable.  They kinda suck.

**Comment by dorothealint** [comment]
It worked now though!  So I believe you have those scripts at your disposal if you need them!  I think that is all of the shorts.

**Comment by helvitiis** [comment]
I have no idea what might have caused that. I looked up the error and here is something I found on Stack Overflow: https://stackoverflow.com/questions/7438313/pushing-to-git-returning-error-code-403-fatal-http-request-failed

It won't help now that you've fixed it but in the future it might!

**Comment by dorothealint** [comment]
I just closed everything and reset the password!  Then I had to enter my username and password in each of my spaces and it pushed everything that was sitting out there waiting in the ether.  I didn't have to re-add anything.

**Comment by helvitiis** [comment]
Well, at least it's working now! I'm looking at the scripts now and laughing. I love how short they are. This is great!

**Comment by dorothealint** [comment]
On a side note...if I were to try to do an xquery on these, the collection would be something like  db/rickandmorty/shortsFiles?

**Comment by dorothealint** [comment]
They are short!  If we get the regular scripts labeled with those attributes, which I'm going to work on over break, we should be able to generate some really nice graphs and tables of characters and time travel!

**Comment by helvitiis** [comment]
This is going to be an awesome project. I'm so hype. And the eXist db collection would be `collection('/db/rickandmorty/shortsFiles')`!
